app/config/collections.php

```diff
@@ -34,28 +34,6 @@
                 'array' => false,
                 'filters' => [],
             ],
-            [
-                '$id' => 'resourceType',
-                'type' => Database::VAR_STRING,
-                'format' => '',
-                'size' => 255,
-                'signed' => true,
-                'required' => false,
-                'default' => null,
-                'array' => false,
-                'filters' => [],
-            ],
-            [
-                '$id' => ID::custom('mimeType'),
-                'type' => Database::VAR_STRING,
-                'format' => '',
-                'size' => 255, // https://tools.ietf.org/html/rfc4288#section-4.2
-                'signed' => true,
-                'required' => false,
-                'default' => null,
-                'array' => false,
-                'filters' => [],
-            ],
```

Comment:
lets keep that for migration in mind :-)

Reply:
There is going to be a follow up PR to add these attributes back. And yes migration is there
https://github.com/appwrite/appwrite/blob/1.5.x/src/Appwrite/Migration/Version/V20.php#L139-L155